Hello world + buble sorting (Task1New)

DaimonV15 opened the pull request.

DaimonV15 committed:
Hello world + buble sorting (Task1New)

xthebat reviewed (commented):
- Удалите бинарные файлы, необходимо оставить только 
```
src/
gradle/
build.gradle
.gitignore
gradlew
gradlew.bat
settings.gradle
```

- Добавить unit-test

DaimonV15 committed:
add unit test

xthebat commented:
- Удаленные бинарные файлы - ок.
- То что Вы написали, не является unit-test (для этого должна быть использована библиотека junit) см. 3-е видео

DaimonV15 committed:
add unit test 2

Весьма большие проблемы с гит хабом, не могу до конца разобраться ка правильно удалять предыдущие файлы и заливать новые. Попробовал добавить gitignore
